client/cli/src/params/keystore_params.rs

```diff
@@ -30,6 +30,9 @@ const DEFAULT_KEYSTORE_CONFIG_PATH: &'static str = "keystore";
 /// Parameters of the keystore
 #[derive(Debug, StructOpt)]
 pub struct KeystoreParams {
+	/// Specify custom URIs to connect to for keystore-services
+	#[structopt(long = "keystore-uri")]
+	pub keystore_uris: Vec<String>,
```

Comment:
> and thus begs the question whether to even implement the CLI interface as that in anticipation or not

Since multiple is not implemented yet, I think you can just make `keystore_uri: String` and put it in a `vec![]` (in keystore_config). That will remove some code and it won't break the compatibility later on because `String` can be parsed to `Vec<String>`.

Though it's not breaking anything so I'm fine if you keep it this way. I'm just afraid of too much anticipation for a feature that never arrives :confounded: